projects/netgrif-components-core/src/lib/view/case-view/abstract-case-view.spec.ts

```diff
@@ -82,10 +89,13 @@ describe('AbstractCaseView', () => {
             expect(component).toBeTruthy();
         });
 
-        it('should display create case button', () => {
+        it('should display create case button', (done) => {
             expect(component).toBeTruthy();
-            expect(allowedNetsService.allowedNets.length).toBe(1);
-            expect(component.hasAuthority()).toBeTrue();
+            caseViewService.getNewCaseAllowedNets().subscribe(nets => {
+                expect(allowedNetsService.allowedNets.length).toBe(1);
```

Comment:
this doesn't test the code in the component but in the service, that should not be the case.

The `component.canCreate` property should be checked instead, so that it will reflect any implementation changes to the check itself

Reply:
updated according to comment